docs/formats.rst

```diff
@@ -44,6 +44,51 @@ Weblate can automatically detect several widely spread file formats, but this
 detection can harm your performance and will limit features specific to given
 file format (for example automatic adding of new translations).
 
+.. _fmt_capabs:
+
+Translation types capabilities
+------------------------------
+
+Below are listed capabilities of all supported formats.
+
++---------------------+------------+----------+---------+----------+
+| Format              | Linguality | Comments | Context | Location |
++=====================+============+==========+=========+==========+
+| :ref:`gettext`      | bilingual  | yes      | yes     | yes      |
++---------------------+------------+----------+---------+----------+
+| :ref:`mono_gettext` | mono       | yes      | yes     | yes      |
++---------------------+------------+----------+---------+----------+
+| :ref:`xliff:        | both       | yes      | yes     |          |
```

Comment:
Typo here, see [failed build on travis](https://travis-ci.org/WeblateOrg/weblate/jobs/425769749):

```
weblate/docs/formats.rst:62:Inline interpreted text or phrase reference start-string without end-string.
```

Reply:
Thank you!

Fixed.